helix-lock/src/main/java/org/apache/helix/lock/HelixLock.java

```diff
@@ -22,7 +22,7 @@
 /**
  * Generic interface for Helix distributed lock
  */
-public interface HelixLock<T> {
+public interface HelixLock {
```

Comment:
Let's have the interface as DistributedLock or something. Name it HelixLock still seemed to be sticking to Helix.